docs/detections/prebuilt-rules/rule-details/potential-dll-side-loading-via-microsoft-antimalware-service-executable.asciidoc

```diff
@@ -0,0 +1,73 @@
+[[potential-dll-side-loading-via-microsoft-antimalware-service-executable]]
+=== Potential DLL Side-Loading via Microsoft Antimalware Service Executable
+
+Identifies a Windows trusted program that is known to be vulnerable to DLL Search Order Hijacking starting after being renamed or from a non-standard path. This is uncommon behavior and may indicate an attempt to evade defenses via side-loading a malicious DLL within the memory space of one of those processes.
```

Comment:
```suggestion
Identifies a Windows trusted program that is known to be vulnerable to Dynamic-Link Library (DLL) Search Order Hijacking starting after being renamed or from a non-standard path. This is uncommon behavior and may indicate an attempt to evade defenses via side-loading a malicious DLL within the memory space of one of those processes.
```

Reply:
DLL is common enough that I think we are ok to bypass spelling it out

Reply:
Works for me! It looked as though most acronyms were defined on their first instance, so I left this comment just in case.